ddl/db_partition_test.go

```diff
@@ -2994,6 +3001,17 @@ func (s *testIntegrationSuite5) TestDropSchemaWithPartitionTable(c *C) {
 	})
 	c.Assert(err, IsNil)
 
+	startTime := time.Now()
+	done := waitGCDeleteRangeDone(c, tk, tableIDs[2])
+	if !done {
+		// Takes too long, give up the check.
+		logutil.BgLogger().Info("drop schema",
+			zap.Int64("id", tableIDs[0]),
+			zap.Stringer("duration", time.Since(startTime)),
```

Comment:
For async test, the wait time is underestimated/uncontrollable when the CI is with a high workload. Better way is to sync it or move it out of unit tests.

Reply:
Agree, in my local env, it's fast and never run here.
When it's slow in CI and fails, the log is helpful for debugging.

Reply:
make sense

Reply:
Can not we set the logging as `Debug` ?

Reply:
Hey, buddy, don't be too serious... it's the test code and never affect the user. @xhebox 
The default loglevel in CI is not set to `Debug`, so if it really happen we will miss the information.